### ISSUE TITLE
Edges using bezier curve 

### ISSUE THREAD

**Issue by popadicbranislav** [dash14/v-network-graph#76]
Enable graphs to use bezier curves as edge style option (now only straight line, if more edges between same nodes it is curved)

**Comment by dash14** [comment]
Hi @popadicbranislav,
Exactly, now if there is only one edge between nodes, it will be a straight line even with the `curve` option.
Just to confirm, is this issue a feature request?

**Comment by popadicbranislav** [comment]
Hi @dash14,
that would be a feature request.

**Comment by tch1001** [comment]
indeed, this would be nice to have! i will see if i can contribute anything! thanks for this library btw :)

**Comment by pavelhoral** [comment]
This might be useful if nodes have defined output and input ports / sides. For example check dialogue graph in this article - https://www.articy.com/en/showcase/disco-elysium/ . In this example the outgoing link is comming out from the right side of the node and enters the left side of next node.

**Comment by IARI** [comment]
being able to somehow customize the edge layout would be amazing
